src/openms/include/OpenMS/ANALYSIS/OPENSWATH/PeakIntegrator.h

```diff
@@ -518,6 +518,15 @@ namespace OpenMS
       const double peak_height, const double peak_apex_pos
     ) const;
 
+    template <typename PeakContainerConstIteratorT>
+    double findPosAtPerc_(
```

Comment:
What is "perc"?  I am guessing that it is percent.  It maybe best just to call it "percent" instead of "perc".

Reply:
It is percent